modules/installation-launching-installer.adoc

```diff
@@ -301,26 +293,25 @@ https://ovirtlab.example.com/ovirt-engine/api
 ----
 endif::openshift-origin[]
 +
-.. For `Is the oVirt CA trusted locally?`, enter `Yes` since you have already set up a CA certificate. Otherwise, enter `No`.
-.. For `oVirt's CA bundle`, if you entered `Yes` for the preceding question, copy the certificate content from `/etc/pki/ca-trust/source/anchors/ca.pem` and paste it here. Then, press `Enter` twice. Otherwise, if you entered `No` for the preceding question, this question does not appear.
-.. For `oVirt engine username`, enter the user name and profile of the {rh-virtualization} administrator using this format:
+.. The installer automatically generates a CA certificate. For `Would you like to use the above certificate to connect to Engine?`, enter `Yes`. Otherwise, enter `No`. If you answer `No`, you need to install {product-title} in insecure mode.
+.. For `Engine username`, enter the user name and profile of the {rh-virtualization} administrator using this format:
 +
 [source,terminal]
 ----
-<username>@<profile> <1>
+$ <username>@<profile> <1>
```

Comment:
Same question here on whether the "$" is actually appropriate or not.

Reply:
Fixed.